runner_service/services/playbook.py

```diff
@@ -215,7 +215,8 @@ def start_playbook(playbook_name, vars=None, filter=None, tags=None):
     # this should just be run_async, using 'run' hangs the root logger output
     # even when backgrounded
     parms = {
-        "private_data_dir": configuration.settings.playbooks_root_dir,
+        "project_dir": os.path.join(configuration.settings.playbooks_root_dir, 'project'),
```

Comment:
I'm not sure I understand, but how do we get here unique directory per ansible-runner invocation? Isn't above directory the same for all playbooks?

Reply:
When we don't use `private_data_dir` the directory is created in /tmp with unique name. https://github.com/ansible/ansible-runner/blob/devel/ansible_runner/utils.py#L151

Reply:
OK, thanks

Reply:
The playbook execution will happen in the tmp dir. All meta data will be used from the project.
I just tested it by using a host in inventory and executing a small playbook on it and went without a problem.